pandas/tests/dtypes/test_dtypes.py

```diff
@@ -115,6 +115,11 @@ def test_construction_from_string(self, dtype):
         with pytest.raises(TypeError, match=msg):
             CategoricalDtype.construct_from_string("foo")
 
+    @pytest.importorskip("pyarrow")
```

Comment:
I don't think this test should be skipped if pyarrow is not installed? Because I understood the issue that checking the dtype equality currently raises if pyarrow is *not* installed, while it should return False instead. 

So both if pyarrow is installed or not, this test should pass and `is_dtype_equal` should return False. 

I think you can simply remove the decorator.

Reply:
Done